src/styles/theme.ts

```diff
@@ -0,0 +1,31 @@
+export const baseTheme = {
+    colors: {
+        // Background
+        mainBg: '#0B82FF',
+        secondBg: '#F1F1F1',
+        inputBg: '#F4F4F5',
+        containerBg: '#ffffff',
+
+        // Text
+        title: '#272C32',
+        text: '#272C32',
+        label: '#999999',
+
+        // Colors
+        primary: '#0B82FF',
+        secondary: '#272C32',
+        success: '#30BF90',
+        danger: '#FF5730',
+
+        // Colors
+        line: '#E8E8E8',
+        white: '#ffffff',
```

Comment:
❗
Кажется, что их можно объединить. И, как уже писал, если какие-то цвета именуешь семантически, то нужно везде этого придерживаться, поэтому `white` тут лишний
```suggestion
        // Colors
        primary: '#0B82FF',
        secondary: '#272C32',
        success: '#30BF90',
        danger: '#FF5730',
        line: '#E8E8E8',
        white: '#ffffff',
```